libs/RobotKit/include/RobotController.h

```diff
@@ -425,6 +425,10 @@ class RobotController : public interface::RobotController
 		}
 
 		if (NOT_is_playing && is_autonomous_mode) {
+			// TODO: Determine behavior
+			// 1. Do nothing - Sleeping duration is for Main Menu + 1 Activity (current implementation)
+			// 2. Restart - Sleeping duration is for 1 Activity
+			// 3. Disable - No sleeping duration
```

Comment:
I would propose to restart timer on magic card available, without distinction, whether it is dice card, number or whatever the card, it restart the timer. If a child have a trouble with an activity, this last can be longer than 10 minutes. The only case of Leka being asleep should be when no card is given to Leka for 10 minutes

Reply:
I agree with that